kfs-core/src/main/java/edu/arizona/kfs/pdp/batch/service/impl/PrepaidChecksServiceImpl.java

```
@@ -0,0 +1,507 @@
+                            } else {
+                                disbTotalMap.put(pg.getDisbursementNbr(), tot.add(pd.getNetPaymentAmount()));
+                            }
+                        }
+                    }
+                }
+            }
+           
+            csvw = new CSVWriter(new FileWriter(getPayeeDetailLogFileName(inputFileName)));
+
+            csvw.writeNext(new String[] {
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_0,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_1,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_2,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_3,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_4,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_5,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_6,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_7,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_8,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_9,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_10,
+            	PdpConstants.PrePaidChecksConstants.PAYEE_DETAIL_TABLE_HEADING_11
+            });
+            
+            for(PaymentGroup pg : prepaidChecksFile.getPaymentGroups()) {
+                for (PaymentDetail pd : pg.getPaymentDetails()) {
+                    csvw.writeNext(getPayeeDetailLine(pg, pd, disbTotalMap));
+                }
+            }
+
+        }
+        
+        catch (IOException e) {
+            LOG.error("writePayeeDetailLog() Cannot write to output file", e);
+            throw new RuntimeException("writePayeeDetailLog() Cannot write to output file", e);
+        }
+        
+        finally {
+            if (csvw != null) {

```

No change requested, but: from 358-363 could be replaced by a one line call to IOUtils.closeQuietly(...). I realize this is a port, just wanted to make it known that if we aren't trying to recover the stream, and the finally-block swallows all exceptions, it's the perfect candidate for this one-liner. (Logically speaking, the same is happening under the hood in that lib, just cleaner to read.)

---

Noted. If If have to go back and make changes, I will clean this up.